packages/web3-react/package.json

```diff
@@ -9,7 +9,12 @@
     "url": "git+https://github.com/alephium/alephium-web3-react"
   },
   "type": "module",
-  "exports": "./build/index.es.js",
+  "exports": {
+    ".": {
+      "types": "./build/index.d.ts",
+      "import": "./build/index.es.js"
```

Comment:
Since we have declared `"type": "module"`, we only need to specify the `import` here.